src/main.rs

```diff
@@ -1,28 +1,49 @@
 #![feature(proc_macro_hygiene, decl_macro)]
 
+use std::collections::HashMap;
+use std::ops::Deref;
+use std::path::{Path, PathBuf};
+use std::sync::Mutex;
+use std::time::Duration;
+use std::{fs, io};
+
 #[macro_use]
 extern crate rocket;
-use rocket::request::FlashMessage;
-use rocket::request::Form;
-use rocket::response::Flash;
-use rocket::response::NamedFile;
-use rocket::response::Redirect;
+use maplit::hashmap;
+use rocket::request::{FlashMessage, Form};
+use rocket::response::{Flash, NamedFile, Redirect};
 use rocket::State;
 use rocket_contrib::templates::Template;
 use serde::Serialize;
-use std::collections::HashMap;
-use std::env;
-use std::fs;
-use std::io;
-use std::path::{Path, PathBuf};
-use std::sync::Mutex;
-use std::time::Duration;
+use structopt::StructOpt;
 
 mod audio_excl;
 use audio_excl::{init_map, FileLister};
 
-struct AudioMX {
-    mp: Mutex<HashMap<String, FileLister>>,
+struct AudioMX(Mutex<HashMap<String, FileLister>>);
```

Comment:
A [tuple struct](https://doc.rust-lang.org/1.9.0/book/structs.html#tuple-structs) seems like a good fit here -- unlike `type AudioMX = ...` we keep type-safety (i.e. an `AudioMX` instance is something *different* at the type-level than just a `Mutex<...>`), but we don't have to worry about the field's name.